src/frontend/Canonicalize.ml

```diff
@@ -141,9 +141,9 @@ and keep_parens {expr; emeta} =
 
 let parens_lval = map_lval_with no_parens ident
 
-let stmt_to_block ({stmt; smeta} : typed_statement) : typed_statement =
+let stmt_to_block ({stmt; smeta} as s : typed_statement) : typed_statement =
   match stmt with
-  | Block _ -> {stmt; smeta}
+  | Block _ -> s
```

Comment:
If you're going to change this then don't bother destructuring `{stmt;smeta}`, just use `s.stmt` and `s.smeta` where appropriate.

Reply:
Reverted this, initially I was doing more in stmt_to_block